### DISCUSSION TITLE
Model Predictive Control with autopilot mode which enables the stability of Crazy-flie

### DISCUSSION THREAD

**Discussion by WiresharkIO** [bitcraze/discussions#527] [General]
I have a functionality based on Model Predictive control(MPC) which determines the position, linear velocities vx, vy, vz and heading angular velocity vphi. 
I want to operate crazyflie in autopilot that controls its stability while I give the velocity commands by the MPC generated values. 

How can i achieve this hover frame in crazyflie that is the autopilot mode which provides me stability with the drone?

**Comment by krichardsson** [comment]
Duplicate of #528
